src/main/java/org/kuali/kfs/module/purap/document/web/struts/PurchasingActionBase.java

```diff
@@ -1658,13 +1660,15 @@ public ActionForward insertBONote(ActionMapping mapping, ActionForm form, HttpSe
 				GlobalVariables.getMessageMap().putError(String.format("%s.%s",KRADConstants.NEW_DOCUMENT_NOTE_PROPERTY_NAME,
 						KRADConstants.NOTE_TOPIC_TEXT_PROPERTY_NAME), CUPurapKeyConstants.ERROR_ADD_NEW_NOTE_SEND_TO_VENDOR_NO_ATT);
 				return mapping.findForward(KFSConstants.MAPPING_BASIC);
-			} else {
-				if (isAttachmentSizeExceedSqLimit(form, "add")) {
-					return mapping.findForward(KFSConstants.MAPPING_BASIC);
-				} else if (attachmentFile.getFileSize() > SIZE_5MB) {
-					GlobalVariables.getMessageMap().putError(String.format("%s.%s",KRADConstants.NEW_DOCUMENT_NOTE_PROPERTY_NAME,
-							KRADConstants.NOTE_TOPIC_TEXT_PROPERTY_NAME), CUPurapKeyConstants.ERROR_ATT_FILE_SIZE_OVER_LIMIT, attachmentFile.getFileName(), "5");
-					return mapping.findForward(KFSConstants.MAPPING_BASIC);					
+            } else {
+                String fileSizeLimitParameterValue = getMaxPoSendToVendorFileSizeParameterValue();
+                if (isAttachmentSizeExceedSqLimit(form, "add")) {
+                    return mapping.findForward(KFSConstants.MAPPING_BASIC);
+                } else if (attachmentFile.getFileSize() > FileUtil.getBytes(fileSizeLimitParameterValue)) {
+                    String errorMessage = String.format("%s.%s",KRADConstants.NEW_DOCUMENT_NOTE_PROPERTY_NAME, KRADConstants.NOTE_TOPIC_TEXT_PROPERTY_NAME);

```

Comment:
This is actually the property that the error message will be tied to; it's not the error message or message key itself. Please rename the variable to `propertyName` or something similar to make that clearer.

Reply:
Okay I have fixed this, good catch thank you